guides/common/modules/con_managing-dhcp-using-smart-proxies.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Managing_DHCP_Using_Smart_Proxies_{context}"]
+= Managing DHCP Using {SmartProxies}
+
+{SmartProxies} can manage DHCP within their network.
+You can use the DHCP module of {SmartProxy} to query for available IP addresses, add new, and delete existing reservations.
+Note that your {SmartProxy} cannot manage subnet declarations.
+ifdef::foreman-el,foreman-deb,katello[]
+For more information on managing subnet declarations, see https://github.com/theforeman/puppet-dhcp[puppet-dhcp].
+endif::[]
+
+The DHCP module can manage various available DHCP providers:
+
+* `dhcp_infoblox` - Infoblox DHCP server
+* `dhcp_isc` - ISC DHCP server over OMAPI
+* `dhcp_remote_isc` - ISC DHCP server over OMAPI with leases mounted through networking
+* `dhcp_libvirt` - dnsmasq DHCP via libvirt API
```

Comment:
```suggestion
ifndef::satellite[]
* `dhcp_libvirt` - dnsmasq DHCP via libvirt API
endif::[]
```

Reply:
Applied for libirt.

dhcp_native_ms is orcharhino only:
```
git grep -i -n -C 1 assembly_configuring-smart-proxy-on-microsoft-windows.adoc
guides/doc-Installing_Proxy/master.adoc-28-ifdef::orcharhino[]
guides/doc-Installing_Proxy/master.adoc:29:include::common/assembly_configuring-smart-proxy-on-microsoft-windows.adoc[leveloffset=+1]
guides/doc-Installing_Proxy/master.adoc-30-endif::[]
```